src/Handlers/CrawlRequestFulfilled.php

```diff
@@ -2,29 +2,12 @@
 
 namespace Spatie\Crawler\Handlers;
 
-use Spatie\Crawler\Crawler;
-use Spatie\Crawler\CrawlUrl;
-use Spatie\Crawler\LinkAdder;
+use Psr\Http\Message\ResponseInterface;
 use Spatie\Crawler\CrawlerRobots;
 use Spatie\Crawler\CrawlSubdomains;
-use Psr\Http\Message\StreamInterface;
-use Psr\Http\Message\ResponseInterface;
 
-class CrawlRequestFulfilled
+class CrawlRequestFulfilled extends CrawlRequestFulfilledAbstract
```

Comment:
I really don't like the `Abstract` suffix. Rename this class `CrawlRequestFulfilled` and add a prefix `Default` to the actual implementation.